pkg/util/oidcbuilder/jwks.go

```diff
@@ -0,0 +1,136 @@
+package oidcbuilder
```

Comment:
No `jwks_test.go`?

Reply:
The majority of the flow was already tested via the oidcbuilder functions, thus kept only a single test file for this package.